android/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.billingclient:billing:1.1'
```

Comment:
It seems you've downgraded `billing` API. Not using above and importing `AIDL` file seperately is deprecated from android iap sdk from v3.

Reply:
Actually, I think we had some miss communication in #73. I think the gradle `dep` should be used without separate importing.